### PR TITLE
[SMALLFIX] [branch-1.3] Fix flakiness of FileSystemMasterIntegrationTest

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

**Comment by ifcharming** [comment]
Tested locally for 10 runs. I'll kick off multiple Jenkins runs to verify.

**Comment by ifcharming** [comment]
Jenkins, test this please

**Comment by ifcharming** [comment]
Jenkins, test this please

**Comment by ifcharming** [comment]
@jsimsa @aaudiber

**Comment by ifcharming** [comment]
Jenkins, test this please

File: tests/src/test/java/alluxio/LocalAlluxioClusterResource.java [hunk 1]
```diff
@@ -129,6 +129,8 @@ public LocalAlluxioClusterResource setProperty(PropertyKey key, Object value) {
   public void start() throws IOException, AlluxioException {
     // Init configuration for integration test
     mLocalAlluxioCluster.initConfiguration();
+    alluxio.security.authentication.AuthenticatedClientUser.remove();
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/LocalAlluxioClusterResource.java]: could we import instead of using absolute paths?

**Comment by jsimsa** [comment]
LGTM, modulo addressing andrew's comment

File: tests/src/test/java/alluxio/LocalAlluxioClusterResource.java (hunk 1 above)
Comment (ifcharming) [inline tests/src/test/java/alluxio/LocalAlluxioClusterResource.java]: done

File: tests/src/test/java/alluxio/LocalAlluxioClusterResource.java [hunk 2]
```diff
@@ -129,6 +131,8 @@ public LocalAlluxioClusterResource setProperty(PropertyKey key, Object value) {
   public void start() throws IOException, AlluxioException {
     // Init configuration for integration test
     mLocalAlluxioCluster.initConfiguration();
+    AuthenticatedClientUser.remove();
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/LocalAlluxioClusterResource.java]: logically the `initConfiguration()` call goes along with the calls to `Configuration.set`, so we should move these resets to the start of the method or to immediately before `start()`

Comment (ifcharming) [inline tests/src/test/java/alluxio/LocalAlluxioClusterResource.java]: done, thanks

**Comment by ifcharming** [comment]
Jenkins, test this please

**Comment by ifcharming** [comment]
Jenkins, test this please

**Pull request merged**